packages/dialtone-icons/src/keywords.json

```diff
@@ -2356,4 +2356,4 @@
       ]
     }
   }
-}
+}
```

Comment:
Remove the line at the end please

Reply:
Sorry seems like its only whitespace changes, not a new line.

Reply:
Yeah! I noticed it had a line and removed it, that's why it's in the pr hah. 😄